assets/css/style.css

```diff
@@ -1,11 +1,18 @@
 .pulse {
+	all: unset;
 	border-radius: 50%;
+	border: 0;
+	display: block;
 	position: absolute;
 	cursor: pointer;
 	width: 12px;
```

Comment:
There's a minimum size for click targets in WCAG of 24px by 24px. Now, this isn't actually required, because the criteria includes an exemption if "the function can be achieved through a different control on the same page that meets this criterion". However, if the admin bar were not present or not accessible via mouse, then this would fail. It would be good to increase the target size a bit, at least, to make it easier for users. See: https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html